MyString.java

```diff
@@ -64,8 +97,13 @@ public static String spacedString(String str) {
      * @return a randomly generated string, consisting of 'n' lowercase letters
      */
     public static String randomStringOfLetters(int n) {
-        //// Replace the following statement with your code
-        return null;
+        String newStr = "";
+        int randChar = 0;
+        for (int i = 0; i < n; i++){
+            randChar = 97 + (int)(26.0 * Math.random());

```

Comment:
1. use int over double
2. int * double = double 
3. 26, 97 magic values, use formula or constant for first, 'a' for second